k8s/production/gitlab/release.yaml

```diff
@@ -85,8 +89,9 @@ spec:
         username: gitlab
 
       redis:
-        password:
-          enabled: true
+        host: <retrieved from TF-managed configmap>
```

Comment:
This has to be defined here to be overridden later?

Reply:
No, I included it as documentation of sorts to make it easier to see the structure of that field without having to look at the TF config map too. If it introduces more confusion I'm open to changing it.

Reply:
I think it would be sufficient to add a comment to the top of the `redis` block, explaining that it's configured via the terraform configmap.

Reply:
Sorry, I meant that we should remove this field and just rely on the comment (since that's essentially what the value of `host` is now).

Reply:
Oh, my bad. That's fine with me, I updated it